abnf/odata-aggregation-abnf.txt

```diff
@@ -130,15 +130,25 @@ pathPrefix      = [ ( qualifiedEntityTypeName / qualifiedComplexTypeName ) "/" ]
 pathSegment     = ( complexProperty / complexColProperty ) [ "/" qualifiedComplexTypeName ]
                 / navigationProperty [ "/" qualifiedEntityTypeName  ]

```

Comment:
Must we extend `pathSegment` in analogy with `grpPropPathSegment`? Are annotations and type casts also allowed in `aggregate(... from ...)` [OData-Aggr, section 3.1.4]?